### ISSUE TITLE
How disable titles and descriptions?

### ISSUE THREAD

**Issue by zlodiak** [Ks89/angular-modal-gallery#179]
please help my hidden all titles and descriptions. i try follow config:
```
images: Image[] = [
  new Image(
    0,
    { // modal
      img: 'https://bla/img1.jpg'
      alt: '' || null,
      description: : '' || null,
      ariaLabel: '' || null
    },
    { // plain
      img: 'https://bla/img1.jpg'
      title: '' || null,
      alt: '' || null,
      description: : '' || null,
      ariaLabel: '' || null
    }
  )
]

```
## Current behavior
when i hover on plain or modal image, then render titles.

## Expected behavior
when i hover on plain or modal image, then do not render titles.

## Minimal reproduction of the problem with instructions
live minimal example is here: https://stackblitz.com/edit/angular-modal-gallery-v5-asmqs5

Browser:
any browser

**Comment by Ks89** [comment]
Hi.
Please check the documentation website https://ks89.github.io/angular-modal-gallery-2018-v7.github.io/.
I created two live examples about this (under example menu) and also full example inside the repo https://github.com/Ks89/angular-modal-gallery/tree/master/src/app/modal-gallery

What you should do is simply pass the right currentimageconfig object to hide descriptions.

**Comment by Ks89** [comment]
Is this still an issue? Do you need more info?

**Comment by zlodiak** [comment]
> Is this still an issue? Do you need more info?

Thank you, i disabled title and desccription in a modal mode. But i do not found setting for disable title for basic previews line.
[https://i.yapx.ru/DN1cg.jpg](url)

This setting is exist?

**Comment by Ks89** [comment]
To remove titles, you shouldn't add title property in image. In fact, it must be undefined and not empty string to be removed.

**Comment by zlodiak** [comment]
> To remove titles, you shouldn't add title property in image. In fact, it must be undefined and not empty string to be removed.

I try do not add titles to image:
```
  images: Image[] = [
    new Image(
      0,
      { img: 'https://bla/img1.jpg'}
    ),
    new Image(
      1,
      { img: 'https://bla/img2.png'}     
    )
  ];
```
But titles do not dissapear. 
Live example here: https://stackblitz.com/edit/angular-modal-gallery-v5-zw4dx2 

Again will tell about problem: when i hover cursor at an image, then a title is render. I need disable anithing title. Sorry for my english)

**Comment by Ks89** [comment]
I'll try again this evening on my pc, because I'm travelling right now.

**Comment by Ks89** [comment]
You are right, It's not possible to disable titles at the moment. I completely forgot how I implemented this.
About ariaLabel is not a good thing to do, because you'll remove a useful feature from the accessibility point of view, and I'm handling this completely for free and automatically.

I cannot add it to 7.3.0, because this could cause a breaking change, so I have to schedule it for 8.0.0, but with low priority, because 8.0.0 it's already big.

Why would you remove titles? Which is the real scenario?

**Comment by Ks89** [comment]
Ah ok. Yes I cannot implement it quickly right now, because I'm really
busy.

Il ven 25 gen 2019, 07:28 Sergey Kalinin <notifications@github.com> ha
scritto:

**Comment by Ks89** [comment]
I'm working on this issue and It's officially scheduled for 8.0.0-beta.1 that is coming soon.

When I'll implement this I'll close this issue. I still need some days to complete everything and commit in branch `800`

**Comment by Ks89** [comment]
Implemented and it's working as expected, simply passing an empty string to title property in image class. For more info see branch `800`.

In next releases I'll improve this adding also default titles to previews (both modal and carousel), because at the moment, if you don't provide a title, previous won't have titles.
I found this issue while working on #179, but it isn't a big problem, so I'll schedule this as a low priority improvement in upcoming versions (major releases because it's a small breaking change).

Issue closed, because done and scheduled for 8.0.0-beta.1 that is coming soon.

thanks for the patience.
